src/metrics.ts

```diff
@@ -60,43 +61,82 @@ export interface SinglePodMetrics extends PodMetric {
     apiVersion: 'metrics.k8s.io/v1beta1';
 }
 
+export interface SingleNodeMetrics extends NodeMetric {
+    kind: 'NodeMetrics';
+    apiVersion: 'metrics.k8s.io/v1beta1';
+}
+
+export interface GetPodMetricsOptions {
```

Comment:
Since this is used for both PodMetrics and NodeMetrics Perhaps rename this to `MetricsOptions` instead of `GetPodMetricsOptions`?